Detections/ASimAuthentication/imAuthSigninsMultipleCountries.yaml

```diff
@@ -36,7 +37,7 @@ entityMappings:
     fieldMappings:
       - identifier: FullName
```

Comment:
FullName is deprecated now. Please use Name and UPNSuffix.
Also, AccountCustomEntity mapping technique is also old, please use the variables directly for Entity mapping.

Reply:
I just reuse the mapping from the library. If you can show me an example on how to do it properly in YAML, i'll add it.

Reply:
Example - https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Azure%20Active%20Directory/Analytic%20Rules/DisabledAccountSigninsAcrossManyApplications.yaml